vendor/vendor.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
govendor.

I tried doing it all at once and it is very difficult. I think if we slowly update the dependencies and track where they came from at what revision we'll get there eventually.